lib/fakefs/base.rb

```diff
@@ -78,18 +94,18 @@ def clear!
     end
 
     # present a fresh new fake filesystem to the block
-    def with_fresh(&block)
+    def with_fresh(io_mocks: false, &block)
```

Comment:
I think this works too:
```suggestion
    def with_fresh(**kwargs, &block)
```

Reply:
It does work but I find it less explicit and in more generic cases it could trigger weird behaviour on the default value, for instance if the default value was `true` 🤷🏻 

wdyt?